WaterLog_Backend/appsettings.json

```diff
@@ -1,9 +1,13 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Warning"
-        }
-    },
-    "AllowedHosts": "*"
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
 
+  "LiveConnectionString": "Data Source=.\\SQLEXPRESS;Initial Catalog=iot;Integrated Security=true",
```

Comment:
Henko told you guys you shouldn't be committing connection strings and passwords, Ivan will have to reset the passwords again

Reply:
i sent it by mistake am gonna take it out now